### ISSUE TITLE
Printing Issues

### ISSUE THREAD

**Issue by cannycookie** [aschmelyun/docker-compose-laravel#181]
Has anyone ever printed the Hitchhikers Guide on a label printer before?  

### Chapter One

The house stood on a slight rise just on the edge of the village. It stood on its own and looked out over a broad spread of West Country farmland. Not a remarkable house by any means—it was about thirty years old, squattish, squarish, made of brick, and had four windows set in the front of a size and proportion which more or less exactly failed to please the eye.

The only person for whom the house was in any way special was Arthur Dent, and that was only because it happened to be the one he lived in. He had lived in it for about three years, ever since he had moved out of London because it made him nervous and irritable. He was about thirty as well, tall, dark-haired and never quite at ease with himself. The thing that used to worry him most was the fact that people always used to ask him what he was looking so worried about. He worked in local radio which he always used to tell his friends was a lot more interesting than they probably thought. It was, too—most of his friends worked in advertising.

On Wednesday night it had rained very heavily, the lane was wet and muddy, but the Thursday morning sun was bright and clear as it shone on Arthur Dent’s house for what was to be the last time.

It hadn’t properly registered yet with Arthur that the council wanted to knock it down and build a bypass instead.

At eight o’clock on Thursday morning Arthur didn’t feel very good. He woke up blearily, got up, wandered blearily round his room, opened a window, saw a bulldozer, found his slippers, and stomped off to the bathroom to wash.

Toothpaste on the brush—so. Scrub.

Shaving mirror—pointing at the ceiling. He adjusted it. For a moment it reflected a second bulldozer through the bathroom window. Properly adjusted, it reflected Arthur Dent’s bristles. He shaved them off, washed, dried and stomped off to the kitchen to find something pleasant to put in his mouth.

Kettle, plug, fridge, milk, coffee. Yawn.

The word bulldozer wandered through his mind for a moment in search of something to connect with.

The bulldozer outside the kitchen window was quite a big one.

He stared at it.

'Yellow,' he thought, and stomped off back to his bedroom to get dressed.

Passing the bathroom he stopped to drink a large glass of water, and another. He began to suspect that he was hung over. Why was he hung over? Had he been drinking the night before? He supposed that he must have been. He caught a glint in the shaving mirror. “Yellow,” he thought, and stomped on to the bedroom.

He stood and thought. The pub, he thought. Oh dear, the pub. He vaguely remembered being angry, angry about something that seemed important. He’d been telling people about it, telling people about it at great length, he rather suspected: his clearest visual recollection was of glazed looks on other people’s faces. Something about a new bypass he’d just found out about. It had been in the pipeline for months only no one seemed to have known about it. Ridiculous. He took a swig of water. It would sort itself out, he’ d decided, no one wanted a bypass, the council didn’t have a leg to stand on. It would sort itself out.

God, what a terrible hangover it had earned him though. He looked at himself in the wardrobe mirror. He stuck out his tongue. 'Yellow,' he thought. The word yellow wandered through his mind in search of something to connect with.

Fifteen seconds later he was out of the house and lying in front of a big yellow bulldozer that was advancing up his garden path.

Mr. L. Prosser was, as they say, only human. In other words he was a carbon-based bipedal life form descended from an ape. More specifically he was forty, fat and shabby and worked for the local council. Curiously enough, though he didn’t know it, he was also a direct male-line descendant of Genghis Khan, though intervening generations and racial mixing had so juggled his genes that he had no discernible Mongoloid characteristics, and the only vestiges left in Mr. L. Prosser of his mighty ancestry were a pronounced stoutness about the tum and a predilection for little fur hats.

He was by no means a great warrior; in fact he was a nervous, worried man. Today he was particularly nervous and worried because something had gone seriously wrong with his job, which was to see that Arthur Dent’s house got cleared out of the way before the day was out.

“Come off it, Mr. Dent,” he said, “you can’t win, you know. You can’t lie in front of the bulldozer indefinitely.” He tried to make his eyes blaze fiercely but they just wouldn’t do it.

Arthur lay in the mud and squelched at him.

“I’m game,” he said, “we’ll see who rusts first.”

“I’m afraid you’re going to have to accept it,” said Mr. Prosser, gripping his fur hat and rolling it round the top of his head; “this bypass has got to be built and it’s going to be built!”

“First I’ve heard of it,” said Arthur, “why’s it got to be built?”

Mr. Prosser shook his finger at him for a bit, then stopped and put it away again.

“What do you mean, why’s it got to be built?” he said. “It’s a bypass. You’ve got to build bypasses.”

Bypasses are devices that allow some people to dash from point A to point B very fast while other people dash from point B to point A very fast. People living at point C, being a point directly in between, are often given to wonder what’s so great about point A that so many people from point B are so keen to get there, and what’s so great about point B that so many people from point A are so keen to get there. They often wish that people would just once and for all work out where the hell they wanted to be.

Mr. Prosser wanted to be at point D. Point D wasn’t anywhere in particular, it was just any convenient point a very long way from points A, B and C. He would have a nice little cottage at point D, with axes over the door, and spend a pleasant amount of time at point E, which would be the nearest pub to point D. His wife of course wanted climbing roses, but he wanted axes. He didn’t know why—he just liked axes. He flushed hotly under the derisive grins of the bulldozer drivers.

He shifted his weight from foot to foot, but it was equally uncomfortable on each. Obviously somebody had been appallingly incompetent and he hoped to God it wasn’t him.

Mr. Prosser said, “You were quite entitled to make any suggestions or protests at the appropriate time, you know.”

“Appropriate time?” hooted Arthur. “Appropriate time? The first I knew about it was when a workman arrived at my home yesterday. I asked him if he’d come to clean the windows and he said no, he’d come to demolish the house. He didn’t tell me straight away of course. Oh no. First he wiped a couple of windows and charged me a fiver. Then he told me.”

“But Mr. Dent, the plans have been available in the local planning office for the last nine months.”

“Oh yes, well, as soon as I heard I went straight round to see them, yesterday afternoon. You hadn’t exactly gone out of your way to call attention to them, had you? I mean, like actually telling anybody or anything.”

“But the plans were on display . . .”

“On display? I eventually had to go down to the cellar to find them.”

“That’s the display department.”

“With a flashlight.”

“Ah, well, the lights had probably gone.”

“So had the stairs.”

“But look, you found the notice, didn’t you?”

“Yes,” said Arthur, “yes I did. It was on display in the bottom of a locked filing cabinet stuck in a disused lavatory with a sign on the door saying ‘Beware of the Leopard.’”

A cloud passed overhead. It cast a shadow over Arthur Dent as he lay propped up on his elbow in the cold mud. It cast a shadow over Arthur Dent’s house. Mr. Prosser frowned at it.

“It’s not as if it’ s a particularly nice house,” he said.

“I’m sorry, but I happen to like it.”

“You’ ll like the bypass.”

“Oh, shut up,” said Arthur Dent. “Shut up and go away, and take your bloody bypass with you. You haven’t got a leg to stand on and you know it.”

Mr. Prosser’s mouth opened and closed a couple of times while his mind was for a moment filled with inexplicable but terribly attractive visions of Arthur Dent’ s house being consumed with fire and Arthur himself running screaming from the blazing ruin with at least three hefty spears protruding from his back. Mr. Prosser was often bothered with visions like these and they made him feel very nervous. He stuttered for a moment and then pulled himself together.

“Mr. Dent,” he said.

“Hello? Yes?” said Arthur.

“Some factual information for you. Have you any idea how much damage that bulldozer would suffer if I just let it roll straight over you?”

### Chapter 24

Silently the aircar coasted through the cold darkness, a single soft glow of light that was utterly alone in the deep Magrathean night. It sped swiftly. Arthur's companion seemed sunk in his own thoughts, and when Arthur tried on a couple of occasions to engage him in conversation again he would simply reply by asking if he was comfortable enough, and then left it at that.

Arthur tried to gauge the speed at which they were travelling, but the blackness outside was absolute and he was denied any reference points. The sense of motion was so soft and slight he could almost believe they were hardly moving at all.

Then a tiny glow of light appeared in the far distance and within seconds had grown so much in size that Arthur realized it was travelling towards them at a colossal speed, and he tried to make out what sort of craft it might be. He peered at it, but was unable to discern any clear shape, and suddenly gasped in alarm as the aircraft dipped sharply and headed downwards in what seemed certain to be a collision course. Their relative velocity seemed unbelievable, and Arthur had hardly time to draw breath before it was all over. The next thing he was aware of was an insane silver blur that seemed to surround him. He twisted his head sharply round and saw a small black point dwindling rapidly in the distance behind them, and it took him several seconds to realize what had happened.

They had plunged into a tunnel in the ground. The colossal speed had been their own relative to the glow of light which was a stationary hole in the ground, the mouth of the tunnel. The insane blur of silver was the circular wall of the tunnel down which they were shooting, apparently at several hundred miles an hour.

He closed his eyes in terror.

After a length of time which he made no attempt to judge, he sensed a slight subsidence in their speed and some while later became aware that they were gradually gliding to a gentle halt.

He opened his eyes again. They were still in the silver tunnel, threading and weaving their way through what appeared to be a crisscross warren of converging tunnels. When they finally stopped it was in a small chamber of curved steel. Several tunnels also had their terminus here, and at the farther end of the chamber Arthur could see a large circle of dim irritating light. It was irritating because it played tricks with the eyes, it was impossible to focus on it properly or tell how near or far it was. Arthur guessed (quite wrongly) that it might be ultra violet.

Slartibartfast turned and regarded Arthur with his solemn old eyes.

"Earthman," he said, "we are now deep in the heart of Magrathea."

"How did you know I was an Earthman?" demanded Arthur.

"These things will become clear to you," said the old man gently, "at least," he added with slight doubt in his voice, "clearer than they are at the moment."

He continued: "I should warn you that the chamber we are about to pass into does not literally exist within our planet. It is a little too ... large. We are about to pass through a gateway into a vast tract of hyperspace. It may disturb you."

Arthur made nervous noises.

Slartibartfast touched a button and added, not entirely reassuringly. "It scares the willies out of me. Hold tight."

The car shot forward straight into the circle of light, and suddenly Arthur had a fairly clear idea of what infinity looked like.

It wasn't infinity in fact. Infinity itself looks flat and uninteresting. Looking up into the night sky is looking into infinity - distance is incomprehensible and therefore meaningless. The chamber into which the aircar emerged was anything but infinite, it was just very very big, so that it gave the impression of infinity far better than infinity itself.

Arthur's senses bobbed and span, as, travelling at the immense speed he knew the aircar attained, they climbed slowly through the open air leaving the gateway through which they had passed an invisible pinprick in the shimmering wall behind them.

The wall.

The wall defied the imagination - seduced it and defeated it. The wall was so paralysingly vast and sheer that its top, bottom and sides passed away beyond the reach of sight. The mere shock of vertigo could kill a man.

The wall appeared perfectly flat. It would take the finest laser measuring equipment to detect that as it climbed, apparently to infinity, as it dropped dizzily away, as it planed out to either side, it also curved. It met itself again thirteen light seconds away. In other words the wall formed the inside of a hollow sphere, a sphere over three million miles across and flooded with unimaginable light.

"Welcome," said Slartibartfast as the tiny speck that was the aircar, travelling now at three times the speed of sound, crept imperceptibly forward into the mindboggling space, "welcome," he said, "to our factory floor."

Arthur stared about him in a kind of wonderful horror. Ranged away before them, at distances he could neither judge nor even guess at, were a series of curious suspensions, delicate traceries of metal and light hung about shadowy spherical shapes that hung in the space.

"This," said Slartibartfast, "is where we make most of our planets you see."

"You mean," said Arthur, trying to form the words, "you mean you're starting it all up again now?"

"No no, good heavens no," exclaimed the old man, "no, the Galaxy isn't nearly rich enough to support us yet. No, we've been awakened to perform just one extraordinary commission for very ... special clients from another dimension. It may interest you ... there in the distance in front of us."

Arthur followed the old man's finger, till he was able to pick out the floating structure he was pointing out. It was indeed the only one of the many structures that betrayed any sign of activity about it, though this was more a sublimal impression than anything one could put one's finger on.

At the moment however a flash of light arced through the structure and revealed in stark relief the patterns that were formed on the dark sphere within. Patterns that Arthur knew, rough blobby shapes that were as familiar to him as the shapes of words, part of the furniture of his mind. For a few seconds he sat in stunned silence as the images rushed around his mind and tried to find somewhere to settle down and make sense.

Part of his brain told him that he knew perfectly well what he was looking at and what the shapes represented whilst another quite sensibly refused to countenance the idea and abdicated responsibility for any further thinking in that direction.

The flash came again, and this time there could be no doubt.

"The Earth ..." whispered Arthur.

"Well, the Earth Mark Two in fact," said Slartibartfast cheerfully. "We're making a copy from our original blueprints."

There was a pause.

"Are you trying to tell me," said Arthur, slowly and with control, "that you originally ... made the Earth?"

"Oh yes," said Slartibartfast. "Did you ever go to a place ... I think it was called Norway?"

"No," said Arthur, "no, I didn't."

"Pity," said Slartibartfast, "that was one of mine. Won an award you know. Lovely crinkly edges. I was most upset to hear about its destruction."

"You were upset!"

"Yes. Five minutes later and it wouldn't have mattered so much. It was a quite shocking cock-up."

"Huh?" said Arthur.

"The mice were furious."

"The mice were furious?"

"Oh yes," said the old man mildly.

"Yes well so I expect were the dogs and cats and duckbilled platypuses, but ..."

"Ah, but they hadn't paid for it you see, had they?"

"Look," said Arthur, "would it save you a lot of time if I just gave up and went mad now?"

For a while the aircar flew on in awkward silence. Then the old man tried patiently to explain.

"Earthman, the planet you lived on was commissioned, paid for, and run by mice. It was destroyed five minutes before the completion of the purpose for which it was built, and we've got to build another one."

Only one word registered with Arthur.

"Mice?" he said.

"Indeed Earthman."

"Look, sorry - are we talking about the little white furry things with the cheese fixation and women standing on tables screaming in early sixties sit coms?"

Slartibartfast coughed politely.

"Earthman," he said, "it is sometimes hard to follow your mode of speech. Remember I have been asleep inside this planet of Magrathea for five million years and know little of these early sixties sit coms of which you speak. These creatures you call mice, you see, they are not quite as they appear. They are merely the protrusion into our dimension of vast hyperintelligent pan- dimensional beings. The whole business with the cheese and the squeaking is just a front."

The old man paused, and with a sympathetic frown continued.

"They've been experimenting on you I'm afraid."

Arthur thought about this for a second, and then his face cleared.

"Ah no," he said, "I see the source of the misunderstanding now. No, look you see, what happened was that we used to do experiments on them. They were often used in behavioural research, Pavlov and all that sort of stuff. So what happened was hat the mice would be set all sorts of tests, learning to ring bells, run around mazes and things so that the whole nature of the learning process could be examined. From our observations of their behaviour we were able to learn all sorts of things about our own ..."

Arthur's voice tailed off.

"Such subtlety ..." said Slartibartfast, "one has to admire it."

"What?" said Arthur.

"How better to disguise their real natures, and how better to guide your thinking. Suddenly running down a maze the wrong way, eating the wrong bit of cheese, unexpectedly dropping dead of myxomatosis, - if it's finely calculated the cumulative effect is enormous."

He paused for effect.

"You see, Earthman, they really are particularly clever hyperintelligent pan-dimensional beings. Your planet and people have formed the matrix of an organic computer running a ten- million-year research programme ...

"Let me tell you the whole story. It'll take a little time."

"Time," said Arthur weakly, "is not currently one of my problems."

### Chapter 25

There are of course many problems connected with life, of which some of the most popular are Why are people born? Why do they die? Why do they want to spend so much of the intervening time wearing digital watches?

Many many millions of years ago a race of hyperintelligent pan- dimensional beings (whose physical manifestation in their own pan-dimensional universe is not dissimilar to our own) got so fed up with the constant bickering about the meaning of life which used to interrupt their favourite pastime of Brockian Ultra Cricket (a curious game which involved suddenly hitting people for no readily apparent reason and then running away) that they decided to sit down and solve their problems once and for all.

And to this end they built themselves a stupendous super computer which was so amazingly intelligent that even before the data banks had been connected up it had started from I think therefore I am and got as far as the existence of rice pudding and income tax before anyone managed to turn it off.

It was the size of a small city.

Its main console was installed in a specially designed executive office, mounted on an enormous executive desk of finest ultramahagony topped with rich ultrared leather. The dark carpeting was discreetly sumptuous, exotic pot plants and tastefully engraved prints of the principal computer programmers and their families were deployed liberally about the room, and stately windows looked out upon a tree-lined public square.

On the day of the Great On-Turning two soberly dressed programmers with brief cases arrived and were shown discreetly into the office. They were aware that this day they would represent their entire race in its greatest moment, but they conducted themselves calmly and quietly as they seated themselves deferentially before the desk, opened their brief cases and took out their leather-bound notebooks.

Their names were Lunkwill and Fook.

For a few moments they sat in respectful silence, then, after exchanging a quiet glance with Fook, Lunkwill leaned forward and touched a small black panel.

The subtlest of hums indicated that the massive computer was now in total active mode. After a pause it spoke to them in a voice rich resonant and deep.

It said: "What is this great task for which I, Deep Thought, the second greatest computer in the Universe of Time and Space have been called into existence?"

Lunkwill and Fook glanced at each other in surprise.

"Your task, O Computer ..." began Fook.

"No, wait a minute, this isn't right," said Lunkwill, worried. "We distinctly designed this computer to be the greatest one ever and we're not making do with second best. Deep Thought," he addressed the computer, "are you not as we designed you to be, the greatest most powerful computer in all time?"

"I described myself as the second greatest," intoned Deep Thought, "and such I am."

Another worried look passed between the two programmers. Lunkwill cleared his throat.

"There must be some mistake," he said, "are you not a greatest computer than the Milliard Gargantubrain which can count all the atoms in a star in a millisecond?"

"The Milliard Gargantubrain?" said Deep Thought with unconcealed contempt. "A mere abacus - mention it not."

"And are you not," said Fook leaning anxiously forward, "a greater analyst than the Googleplex Star Thinker in the Seventh Galaxy of Light and Ingenuity which can calculate the trajectory of every single dust particle throughout a five-week Dangrabad Beta sand blizzard?"

"A five-week sand blizzard?" said Deep Thought haughtily. "You ask this of me who have contemplated the very vectors of the atoms in the Big Bang itself? Molest me not with this pocket calculator stuff."

The two programmers sat in uncomfortable silence for a moment. Then Lunkwill leaned forward again.

"But are you not," he said, "a more fiendish disputant than the Great Hyperlobic Omni-Cognate Neutron Wrangler of Ciceronicus 12, the Magic and Indefatigable?"

"The Great Hyperlobic Omni-Cognate Neutron Wrangler," said Deep Thought thoroughly rolling the r's, "could talk all four legs off an Arcturan MegaDonkey - but only I could persuade it to go for a walk afterwards."

"Then what," asked Fook, "is the problem?"

"There is no problem," said Deep Thought with magnificent ringing tones. "I am simply the second greatest computer in the Universe of Space and Time."

"But the second?" insisted Lunkwill. "Why do you keep saying the second? You're surely not thinking of the Multicorticoid Perspicutron Titan Muller are you? Or the Pondermatic? Or the ..."

Contemptuous lights flashed across the computer's console.

"I spare not a single unit of thought on these cybernetic simpletons!" he boomed. "I speak of none but the computer that is to come after me!"

Fook was losing patience. He pushed his notebook aside and muttered, "I think this is getting needlessly messianic."

"You know nothing of future time," pronounced Deep Thought, "and yet in my teeming circuitry I can navigate the infinite delta streams of future probability and see that there must one day come a computer whose merest operational parameters I am not worthy to calculate, but which it will be my fate eventually to design."

Fook sighed heavily and glanced across to Lunkwill.

"Can we get on and ask the question?" he said.

Lunkwill motioned him to wait.

"What computer is this of which you speak?" he asked.

"I will speak of it no further in this present time," said Deep Thought. "Now. Ask what else of me you will that I may function. Speak."

They shrugged at each other. Fook composed himself.

"O Deep Thought Computer," he said, "the task we have designed you to perform is this. We want you to tell us ..." he paused, "... the Answer!"

"The answer?" said Deep Thought. "The answer to what?"

"Life!" urged Fook.

"The Universe!" said Lunkwill.

"Everything!" they said in chorus.

Deep Thought paused for a moment's reflection.

"Tricky," he said finally.

"But can you do it?"

Again, a significant pause.

"Yes," said Deep Thought, "I can do it."

"There is an answer?" said Fook with breathless excitement."

"A simple answer?" added Lunkwill.

"Yes," said Deep Thought. "Life, the Universe, and Everything. There is an answer. But," he added, "I'll have to think about it."

A sudden commotion destroyed the moment: the door flew open and two angry men wearing the coarse faded-blue robes and belts of the Cruxwan University burst into the room, thrusting aside the ineffectual flunkies who tried to bar their way.

"We demand admission!" shouted the younger of the two men elbowing a pretty young secretary in the throat.

"Come on," shouted the older one, "you can't keep us out!" He pushed a junior programmer back through the door.

"We demand that you can't keep us out!" bawled the younger one, though he was now firmly inside the room and no further attempts were being made to stop him.

"Who are you?" said Lunkwill, rising angrily from his seat. "What do you want?"

"I am Majikthise!" announced the older one.

"And I demand that I am Vroomfondel!" shouted the younger one.

Majikthise turned on Vroomfondel. "It's alright," he explained angrily, "you don't need to demand that."

"Alright!" bawled Vroomfondel banging on an nearby desk. "I am Vroomfondel, and that is not a demand, that is a solid fact! What we demand is solid facts!"

"No we don't!" exclaimed Majikthise in irritation. "That is precisely what we don't demand!"

Scarcely pausing for breath, Vroomfondel shouted, "We don't demand solid facts! What we demand is a total absence of solid facts. I demand that I may or may not be Vroomfondel!"

"But who the devil are you?" exclaimed an outraged Fook.

"We," said Majikthise, "are Philosophers."

"Though we may not be," said Vroomfondel waving a warning finger at the programmers.

"Yes we are," insisted Majikthise. "We are quite definitely here as representatives of the Amalgamated Union of Philosophers, Sages, Luminaries and Other Thinking Persons, and we want this machine off, and we want it off now!"

"What's the problem?" said Lunkwill.

"I'll tell you what the problem is mate," said Majikthise, "demarcation, that's the problem!"

"We demand," yelled Vroomfondel, "that demarcation may or may not be the problem!"

"You just let the machines get on with the adding up," warned Majikthise, "and we'll take care of the eternal verities thank you very much. You want to check your legal position you do mate. Under law the Quest for Ultimate Truth is quite clearly the inalienable prerogative of your working thinkers. Any bloody machine goes and actually finds it and we're straight out of a job aren't we? I mean what's the use of our sitting up half the night arguing that there may or may not be a God if this machine only goes and gives us his bleeding phone number the next morning?"

"That's right!" shouted Vroomfondel, "we demand rigidly defined areas of doubt and uncertainty!"

Suddenly a stentorian voice boomed across the room.

"Might I make an observation at this point?" inquired Deep Thought.

"We'll go on strike!" yelled Vroomfondel.

"That's right!" agreed Majikthise. "You'll have a national Philosopher's strike on your hands!"

The hum level in the room suddenly increased as several ancillary bass driver units, mounted in sedately carved and varnished cabinet speakers around the room, cut in to give Deep Thought's voice a little more power.

"All I wanted to say," bellowed the computer, "is that my circuits are now irrevocably committed to calculating the answer to the Ultimate Question of Life, the Universe, and Everything -" he paused and satisfied himself that he now had everyone's attention, before continuing more quietly, "but the programme will take me a little while to run."

Fook glanced impatiently at his watch.

"How long?" he said.

"Seven and a half million years," said Deep Thought.

Lunkwill and Fook blinked at each other.

"Seven and a half million years ...!" they cried in chorus.

"Yes," declaimed Deep Thought, "I said I'd have to think about it, didn't I? And it occurs to me that running a programme like this is bound to create an enormous amount of popular publicity for the whole area of philosophy in general. Everyone's going to have their own theories about what answer I'm eventually to come up with, and who better to capitalize on that media market than you yourself? So long as you can keep disagreeing with each other violently enough and slagging each other off in the popular press, you can keep yourself on the gravy train for life. How does that sound?"

The two philosophers gaped at him.

"Bloody hell," said Majikthise, "now that is what I call thinking. Here Vroomfondel, why do we never think of things like that?"

"Dunno," said Vroomfondel in an awed whisper, "think our brains must be too highly trained Majikthise."

So saying, they turned on their heels and walked out of the door and into a lifestyle beyond their wildest dreams.

### Chapter 26
"Yes, very salutary," said Arthur, after Slartibartfast had related the salient points of the story to him, "but I don't understand what all this has got to do with the Earth and mice and things."

"That is but the first half of the story Earthman," said the old man. "If you would care to discover what happened seven and a half millions later, on the great day of the Answer, allow me to invite you to my study where you can experience the events yourself on our Sens-O-Tape records. That is unless you would care to take a quick stroll on the surface of New Earth. It's only half completed I'm afraid - we haven't even finished burying the artificial dinosaur skeletons in the crust yet, then we have the Tertiary and Quarternary Periods of the Cenozoic Era to lay down, and ..."

"No thank you," said Arthur, "it wouldn't be quite the same."

"No," said Slartibartfast, "it won't be," and he turned the aircar round and headed back towards the mind-numbing wall.

**Comment by aschmelyun** [comment]
<3
